Library/ViewModels/BackerDashboardViewModel.swift

```diff
@@ -0,0 +1,239 @@
+import KsApi
+import Prelude
+import ReactiveSwift
+import Result
+
+public protocol BackerDashboardViewModelInputs {
+  /// Call when backed projects button is tapped.
+  func backedProjectsButtonTapped()
+
+  /// Call when messages button is tapped.
+  func messagesButtonTapped()
+
+  /// Call when a project cell is tapped.
+  func projectTapped(_ project: Project)
+
+  /// Call when saved projects button is tapped.
+  func savedProjectsButtonTapped()
+
+  /// Call when settings button is tapped.
+  func settingsButtonTapped()
+
+  /// Call when the project navigator has transitioned to a new project with its index.
+  func transitionedToProject(at row: Int, outOf totalRows: Int)
+
+  /// Call when the view loads.
+  func viewDidLoad()
+
+  /// Call when the view will appear.
+  func viewWillAppear(_ animated: Bool)
+}
+
+public protocol BackerDashboardViewModelOutputs {
+  /// Emits a URL for the avatar image view.
+  var avatarURL: Signal<URL?, NoError> { get }
+
+  /// Emits an attributed string for the backed button title label.
+  var backedButtonTitleText: Signal<String, NoError> { get }
+
+  /// Emits a string for the backer location label.
+  var backerLocationText: Signal<String, NoError> { get }
+
+  /// Emits a string for the backer name label.
+  var backerNameText: Signal<String, NoError> { get }
+
+  /// Emits a boolean whether backed projects container should be hidden or not.
+  var backedProjectsAreHidden: Signal<Bool, NoError> { get }
```

Comment:
not sure i understand this output. what's the container? is it just the line under the button or more? the output should maybe be renamed